spmd/tensor/ops/matrix_ops.py

```diff
@@ -16,17 +16,18 @@ def dist_addmm(
     input: Tensor,
     mat1: Tensor,
     mat2: Tensor,
-    # pyre-fixme[2]: Parameter must be annotated.
-    beta=1,
-    # pyre-fixme[2]: Parameter must be annotated.
-    alpha=1,
+    beta: float = 1,
+    alpha: float = 1,
 ) -> Tensor:
     # dist addmm:
+    # input:replicate   mat1: replicate,  mat2: replicate
```

Comment:
this should not goes into this function, and instead handled by the default handler inside torch_dispatch?